### DISCUSSION TITLE
Doc.ents not working correctly in ver3.3.0

### DISCUSSION THREAD

**Discussion by masoodimran** [explosion/spaCy#10856] [Help: Coding & Implementations]
## How to reproduce the behaviour

doc = nlp("President Trump visited Mexico City.")
doc.ents
displacy.render(doc, style='ent')

![image](https://user-images.githubusercontent.com/30685384/170304248-58e9181a-b623-4d61-b4f2-eea24dcca32e.png)
![image](https://user-images.githubusercontent.com/30685384/170304903-2dfcefae-59ef-45aa-a4b6-d98eeea8c41a.png)

## Info about spaCy

- **spaCy version:** 3.3.0
- **Platform:** macOS-10.16-x86_64-i386-64bit
- **Python version:** 3.8.13
- **Pipelines:** en_core_web_md (3.3.0)

**Comment by polm** [comment]
This is not a bug, the model makes mistakes sometimes, see #3052. 

Note that the training data for the model includes a lot of newspaper texts, but it predates the Trump presidency, so it will not be very sensitive to his name.

**Comment by masoodimran** [comment]
@polm Thanks for the response. I agree to your point regarding the first example in my comment. But, the second one is from the Spacy documentation. How can this not work?

![image](https://user-images.githubusercontent.com/30685384/170412219-36790a03-4971-4a7e-a4f2-e1af19408828.png)

**Reply by polm** [reply]
First, please stop posting screenshots of text. Paste text or code as text. 

What do you expect the output to be there? It looks fine to me. 

Where in the documentation did you see it? Note that we do not test every example in the documentation, and there is no guarantee there will not be changes in the results, for reasons explained in #3052.

**Reply by masoodimran** [reply]
Yes, I know it is correct. What I am trying to say is that it identifies the Person ent correctly in version 2.3.2 and not in 3.3.0. Screenshots because a picture is worth 1000 words. If you need plain text, why provide an option to paste images?

**Reply by masoodimran** [reply]
refer https://spacy.io/api/doc. Section Doc.ents. I have not pasted the screenshot ;-).

**Reply by polm** [reply]
We can't disable pasting screenshots because it's a basic Github feature. In some cases, like with rendering issues related to displaCy, it's the best way to share information.

We ask that code be pasted as text for many reasons. One is that text in images can't be searched for. Another is that if we need to run code for debugging purposes, we can't copy/paste from an image. There are more reasons and these have been discussed abundantly elsewhere, [this Stack Overflow FAQ](https://meta.stackoverflow.com/questions/285551/why-should-i-not-upload-images-of-code-data-errors-when-asking-a-question) may be a useful reference.

**Comment by masoodimran** [comment]
@polm Also, why does it work in this version of Spacy?

![image](https://user-images.githubusercontent.com/30685384/170413973-87b0c98a-a324-45cf-9015-b63f76e26f41.png)

**Reply by polm** [reply]
It's hard to explain why individual predictions turn out the way they do. I don't know why that particular case is different.
